accounts-db/src/accounts_file.rs

```diff
@@ -166,6 +231,11 @@ impl AccountsFile {
     ) -> Option<Vec<StoredAccountInfo>> {
         match self {
             Self::AppendVec(av) => av.append_accounts(accounts, skip),
+            // Currently we only support HOT_FORMAT.  If we later want to use
+            // a different format, then we will need a way to pass-in it.
+            // TODO: consider adding function like write_accounts_to_hot_storage() or something
+            // to hide implementation detail.
+            Self::TieredStorage(ts) => ts.write_accounts(accounts, skip, &HOT_FORMAT).ok(),
```

Comment:
Addressed in https://github.com/anza-xyz/agave/pull/382 by adding TieredStorage::write_accounts_to_hot_storage() and hide the HOT_FORMAT inside.

Reply:
Abandoned #382